openc3/python/test/models/test_metric_model.py

```diff
@@ -0,0 +1,89 @@
+
+# Copyright 2022 Ball Aerospace & Technologies Corp.
```

Comment:
No new files should be copyright Ball Aerospace.   Please update to OpenC3, Inc.

Reply:
Fixed.  The python converter carries the old (C) along with it.